event_model/tests/test_em.py

```diff
@@ -107,11 +107,10 @@ def test_compose_stream_resource(tmp_path):
     compose_stream_resource = bundle.compose_stream_resource
     assert bundle.compose_stream_resource is compose_stream_resource
     bundle = compose_stream_resource(
-        spec="TIFF_STREAM",
-        root=str(tmp_path),
+        mimetype="application/x-hdf5",
```

Comment:
```suggestion
        mimetype="image/tiff",
```

nit but it'd be nice to preserve previous type to show migration path

Reply:
I believe this is valid for a tiff stream with multiple frames?